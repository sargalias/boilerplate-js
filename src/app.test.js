import React from 'react';
import { shallow } from 'enzyme';
import App from './app';

test('test', () => {
  expect(1 + 1).toBe(2);
});

test('snapshot', () => {
  expect([1, 2]).toMatchSnapshot();
});

test('snapshot app', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
