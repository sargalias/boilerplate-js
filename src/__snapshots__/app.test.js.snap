// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Array [
  1,
  2,
]
`;

exports[`snapshot app 1`] = `
<h1
  className="sampleClass"
>
  Hello World!
</h1>
`;
